src/TwigComponent/tests/Integration/ComponentExtensionTest.php

```diff
@@ -208,6 +208,14 @@ public function testComponentPropsOverwriteContextValue(): void
         $this->assertStringContainsString('<p>foo</p>', $output);
     }
 
+    public function testComponentPropsWithTrailingComa(): void
+    {
+        $output = self::getContainer()->get(Environment::class)->render('anonymous_component_props_trailing_coma.html.twig');
```

Comment:
```suggestion
    public function testComponentPropsWithTrailingComma(): void
    {
        $output = self::getContainer()->get(Environment::class)->render('anonymous_component_props_trailing_comma.html.twig');
```

There are more places with this typo.

Reply:
Fixed, thank you 🥲